lib/spoom/cli/commands/run.rb

```diff
@@ -0,0 +1,80 @@
+# typed: true
+# frozen_string_literal: true
+
+require_relative "base"
+
+module Spoom
+  module Cli
+    module Commands
+      class Run < Base
+        default_task :tc
+
+        desc "tc", "run srb tc"
+        option :limit, type: :numeric, aliases: :l
+        option :code, type: :numeric, aliases: :c
+        option :sort, type: :string, aliases: :s
+        def tc
+          in_sorbet_project!
+
+          limit = options[:limit]
+          sort = options[:sort]
+          code = options[:code]
+          colors = !options[:no_color]
+
+          unless limit || code || sort
+            return Spoom::Sorbet.srb_tc(capture_err: false).last
+          end
+
+          output, status = Spoom::Sorbet.srb_tc(capture_err: true)
+          if status
+            $stderr.print(output)
+            return 0
+          end
+
+          errors = Spoom::Sorbet::Errors::Parser.parse_string(output)
+          all_errors = errors.size
```

Comment:
Consider renaming `all_errors` to `errors_count` or similar to make it clear what the variable represents?